src/translations/en-US.json

```diff
@@ -52,5 +52,10 @@
 	"orc-shared.valueName": "Value Name",
 	"orc-shared.yes": "Yes",
 	"orc-shared.embedded": "Embedded",
-	"orc-shared.shared": "Shared"
+	"orc-shared.shared": "Shared",
+	"orc-shared.dataTypeText": "Text",
+	"orc-shared.dataTypeBoolean": "Yes/No Choice",
+	"orc-shared.dataTypeDate": "Date/Calendar",
+	"orc-shared.dataTypeDecimal": "Decimal Number",
+	"orc-shared.dataTypeInteger": "Whole Number"
 }
```

Comment:
add more text for other possible types will be present in custom attributes:
dataTypeEntityReference:  Profile Reference, FR Profil de référence
dataTypeLookup: Lookup List,  FR: Index des listes